### ISSUE TITLE
Accessibility: Missing heading tag [Product Page]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51905]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

This content is not tagged as an HTML heading, though it is one. If content is not appropriately tagged as headings, it will not show in the headings list for screen reader users. Additionally, it may make it more difficult for blind and visually impaired users to understand the structure of the page or to jump to this section.  

![Image](https://github.com/user-attachments/assets/7efe59f9-bf82-427c-8517-0dc846deff0b)

### Component or Page: 

Page: Product Single (Simple)  

### Name affected element:

"Add a review" form heading

### Relevant code snippets:

```
<span id=""reply-title"" class=""comment-reply-title"">Add a review <small><a rel=""nofollow"" id=""cancel-comment-reply-link"" href=""/product/intelligent-paper-knife/#respond"" style=""display:none;"">Cancel reply</a></small></span>

<span id=""reply-title"" class=""comment-reply-title"">Be the first to review “Synergistic Wooden Clock” <small><a rel=""nofollow"" id=""cancel-comment-reply-link"" href=""/product/synergistic-wooden-clock/#respond"" style=""display:none;"">Cancel reply</a></small></span>
```

### Recommended fix:

Wrap content that serves as a title or subtitle with a heading tag. The recommended heading level for this content is: H3.

It is less necessary to have the ""Be the first to review “Product Title”"" text as a heading (the text that shows when there are no reviews yet. But when there are many reviews it is important that the ""Add a review"" text be marked up as a heading so the form can easily be found. You can either make both an H3 (if that's easier) or just mark up the ""Add a review"" text as an H3.
